### PR TITLE
Created default style file to make message styles more dry. Fixes #494.

### LINKED ISSUE #494: [Message] Refactor, may need to be split
This component has become *huge* and is difficult to maintain. May need to be split in different components for each message type

### PR DESCRIPTION
**Pull request by IanCStewart**

### CODE REVIEW

File: src/message/giphy-message/styles.js [hunk 1]
```diff
@@ -41,13 +19,8 @@ export default {
...
-  },
-  iconMenu: {
-    position: 'absolute',
-    top: '3px',
-    right: '0'
+    display: 'inital'
   }
 };
+
+export default merge({}, defaultStyles, overrideStyles);
```
Comment (sjaakluthart) [inline src/message/giphy-message/styles.js]: Do you have a specific reason for using merge? Couldn't this be achieved by using our combineStyles function? Or doesn't that work with nested objects?

Comment (IanCStewart) [reply src/message/giphy-message/styles.js]: Our combineStyle didn't return as was needed. It probably isn't built for nested objects.

Comment (IanCStewart) [reply src/message/giphy-message/styles.js]: merge:```
avatar: {marginLeft: "48px"}
body: {color: "#212121", fontSize: "16px", lineHeight: "22px", margin: "0", maxWidth: "40em"}
compact: {marginLeft: "0", marginRight: "0", maxWidth: "calc(100% - 40px)", display: "flex"}
iconMenu: {position: "absolute", top: "3px", right: "0"}
link: {color: "inherit", fontSize: "inherit", fontWeight: "inherit", textDecoration: "underline"}
mention: {color: "#1BA6C4", fontSize: "inherit", fontWeight: "bolder"}
message: {backgroundColor: "#FEFEFE", borderRadius: "3px", boxSizing: "border-box", color: "#212121", marginLeft: "16px", …}
myAvatar: {marginLeft: "0", marginRight: "48px"}
myMessage: {backgroundColor: "#1BA6C4", color: "#FEFEFE", marginLeft: "0", marginRight: "16px"}
```

combineStyles (note the difference within body):```
avatar: {marginLeft: "48px"}
body: {maxWidth: "40em"}
compact: {maxWidth: "calc(100% - 40px)"}
iconMenu: {position: "absolute", top: "3px", right: "0"}
link: {color: "inherit", fontSize: "inherit", fontWeight: "inherit", textDecoration: "underline"}
mention: {color: "#1BA6C4", fontSize: "inherit", fontWeight: "bolder"}
message: {backgroundColor: "#FEFEFE", borderRadius: "3px", boxSizing: "border-box", color: "#212121", marginLeft: "16px", …}
myAvatar: {marginLeft: "0", marginRight: "48px"}
myMessage: {backgroundColor: "#1BA6C4", color: "#FEFEFE", marginLeft: "0", marginRight: "16px"}
```

Comment (sjaakluthart) [reply src/message/giphy-message/styles.js]: All right, then it's fine! I'll merge.

**Pull request merged**